src/main/scala/path/Path.scala

```diff
@@ -16,6 +18,9 @@ import com.originate.scalypher.where.Where
 
 case class Path(start: Node, pieces: Seq[PathPiece] = Seq.empty) extends Referenceable {
 
+  def set(assignment: Assignment, rest: Assignment*): SetQuery =
```

Comment:
it's unclear if varargs is the way to go here (and in a few other places in this PR), but i like enforcing that at least one assignment exists.